server.js

```diff
@@ -0,0 +1,26 @@
+const express = require('express');
+const mongoose = require('mongoose');
+const bodyParser = require('body-parser');
+const cors = require('cors');
+
+const app = express();
+app.use(bodyParser.json());
+app.use(cors());
+
+mongoose.connect('mongodb://localhost:27017/gaming-wiki', {
```

Comment:
This will need a database to connect to. You can use [docker compose](https://docs.docker.com/compose/compose-application-model/) to run this.